app/services/idv/steps/document_capture_step.rb

```diff
@@ -2,18 +2,28 @@ module Idv
   module Steps
     class DocumentCaptureStep < DocAuthBaseStep
       def call
-        response = post_images
-        if response.success?
+        api_upload = flow_session['api_upload']
+        response = (api_upload && api_upload['documents']) || post_form_images
```

Comment:
We can take advantage of `Hash#dig` here I think:

```suggestion
        response = flow_session.dig('api_upload', 'documents') || post_form_images
```